pkg/network/sriov/sriov_test.go

```diff
@@ -0,0 +1,313 @@
+/*
+* This file is part of the KubeVirt project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Copyright 2022 Red Hat, Inc.
+*
+ */
+
+package sriov_test
+
+import (
+	"kubevirt.io/kubevirt/pkg/network/sriov"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	virtv1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("SRIOV", func() {
+	nonValidNetworkStatusAnnotationValue := ``
```

Comment:
nit: any specific reason to use back-ticks here?
Please consider using:
```go
var nonValidNetworkStatusAnnotationValue string
```

Which makes your intention to use an empty string explicit.

Reply:
back-ticks are used in these strings because they already have "" in the string body. This variable is the exception since it's empty but I think it can stay like that for conformity...

Reply:
It is a nit, but it is indeed odd and the reasoning to have it with "`" is not that strong.
I would even argue that it should be a const.

Reply:
I re-reviewed it and decided to use the explicit `` instead in the function , and doing that I found a typo [here](https://github.com/kubevirt/kubevirt/pull/8226/files#diff-ef56a1a0d52cf224e4e823c30970416c4ad2bd86b85797f5debc316033b3b669R192) so thanks!